Generators/Svg.Generators.csproj

```diff
@@ -8,6 +8,15 @@
         <Nullable>enable</Nullable>
         <LangVersion>latest</LangVersion>
         <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+        <Configurations>Debug;Release</Configurations>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
```

Comment:
Please can you elaborate? Cannot understand what you meant.

Reply:
Is there a way to avoid `CS0618` ?

Reply:
> Is there a way to avoid CS0618 ?

Maybe stop marking codes as `obsolete`. Better still, state the version by which such will be removed,
and then remove the `obsolete` codes and tests accordingly.

On the library side (SVG-NET), we do not need the warnings. 
The end-user needs the warning and should care.

Reply:
The warning was showing up in the workflow tests, so I turned it off in the projects.

Reply:
I don't know how many warnings will show, but should we hide them ?

I think this is information that developers should know.

Reply:
Having moved the tests to Release configuration, we can release the warning in the Debug mode.
Knowing is not sufficient, there must be a plan and documentation of obsolete codes.